app/index.js

```diff
@@ -55,8 +55,6 @@ const isDev = require('electron-is-dev');
 const AppMenu = require('./menus/menu');
 const config = require('./config');
 
-app.commandLine.appendSwitch('js-flags', '--harmony-async-await');
```

Comment:
With this removed, I assume electron runs node 8 now? then we should change travis to reflect the same node version (so that tests run the same node version)  (...for when we have tests 😄 )

Reply:
Yup. Done! e0bb8b7